src/config.js

```diff
@@ -107,6 +107,7 @@ c3_chart_internal_fn.getDefaultConfig = function () {
         axis_x_tick_rotate: 0,
         axis_x_tick_outer: true,
         axis_x_tick_multiline: true,
+        axis_x_tick_multiline_max: 0,
```

Comment:
I think this should be `axis_x_tick_multilineMax` because `axis_x_tick_multiline` is already boolean and settings `axis_x_tick_multiline_max` number makes the type of `axis_x_tick_multiline` ambiguous.

So the

Reply:
Applied suggested changes.